switchboard/switchboard/Sec-Todo.txt

```diff
@@ -0,0 +1 @@
+- Origin & Referer checking
```

Comment:
Might be better to keep this in a tracking issue on GitHub?

Reply:
Created #13.